src/main/java/gregtech/common/covers/VoidingMode.java

```diff
@@ -7,7 +7,7 @@
 public enum VoidingMode implements IStringSerializable {
 
     VOID_ANY("cover.voiding.voiding_mode.void_any", 1),
```

Comment:
Should this also use max int?

Reply:
it could, but it would be strange to set values > 1 on a filter for it to have no effect, since `VOID_ANY` would show a separate text field for numbers